### PR TITLE
Changes to secure Admin Authentication 

### PR DESCRIPTION
**Pull request by mriccell**
Changes to secure Admin username/password

@rosemarymarano @Djelibeybi @TheFrogPad

### CODE REVIEW

**Comment by TheFrogPad** [comment]
Hi @mriccell , the changes look good and are consistent with that resolution to the issue. -Craig

File: OracleWebLogic/dockerfiles/12.2.1.3/README.md [hunk 1]
```diff
@@ -1,105 +1,86 @@
...
+  2. Verify that you now have this image in place with:
 
-### Write your own Oracle WebLogic Server domain with WLST
-The best way to create your own, or extend domains, is by using the [WebLogic Scripting Tool](https://docs.oracle.com/middleware/1221/cross/wlsttasks.htm). You can find an example of a WLST script to create domains at [`create-wls-domain.py`](dockerfiles/12.2.1.3/container-scripts/create-wls-domain.py). You may want to tune this script with your own setup to create datasources and connection pools, security realms, deploy artifacts, and so on. You can also extend images and override an existing domain, or create a new one with WLST.
+        `$ docker images`
 
-## Building the Oracle WebLogic Server Docker image
-To try a sample of a WebLogic Server image with a base domain configured, follow the steps below:
+### Running a Single Server Domain from the image
+The WebLogic Server install image (built above) allows you to run a container with a single WebLogic server domain.  This makes it extreemly simple to deploy applications and any resource the application might need.
```
Comment (rosemarymarano) [inline OracleWebLogic/dockerfiles/12.2.1.3/README.md]: typo "extremely"

File: OracleWebLogic/dockerfiles/12.2.1.3/README.md [hunk 2]
```diff
@@ -1,105 +1,86 @@
...
 
-  1. To build the `12.2.1.3`image, from `dockerfiles`, call:
+#### Providing Admin server Usernasme and Password 
+The username and password must be supplied in a domain.properties file located in a HOST directory that you will map at Docker run time with a -v option. The properties file enables the scripts to configure the correct authentication for the WebLogic Admin server.
```
Comment (rosemarymarano) [inline OracleWebLogic/dockerfiles/12.2.1.3/README.md]: WebLogic Administration Server

File: OracleWebLogic/dockerfiles/12.2.1.3/README.md [hunk 3]
```diff
@@ -1,105 +1,86 @@
...
-  4. Run the Administration Console:
+	$docker run -d -p 7001:7001 -p 9002:9002  -v `HOST PATH where the domain.properties file is`:/u01/oracle/properties -e ADMINISTRATION_PORT_ENABLED=true -e DOMAIN_HOME=/u01/oracle/user_projects/domains/abc_domain -e DOMAIN_NAME=abc_domain oracle/weblogic:12.2.1.3-developer
 
-        `$ docker inspect --format '{{.NetworkSettings.IPAddress}}' <container-name>`
+Run the Administration Console:
 
-        This returns the IP address of the container (for example, `xxx.xx.x.x`). Go to your browser and enter `http://xxx.xx.x.x:7001/console`
+        $ docker inspect --format '{{.NetworkSettings.IPAddress}}' <container-name>
 
+Go to your browser and enter `https://xxx.xx.x.x:9002/console` your browser will request for you to accept Security Exception. To avoid the Security Exception you must update the WebLogic server SSL configuration with a custom identity certificate.
```
Comment (rosemarymarano) [inline OracleWebLogic/dockerfiles/12.2.1.3/README.md]: ...enter `https://xxx.xx.x.x:9002/console`. Your browser will request for you to accept the Security Exception.

**Pull request merged**